packages/gatsby/cache-dir/gatsby-browser-entry.js

```diff
@@ -82,6 +86,8 @@ function graphql() {
 }
 
 export {
+  Img,
+  ImgWithIEPolyfill,
```

Comment:
Does this tree shake?

Reply:
Good question, let me check!

Reply:
I don't think it does, maybe webpack is smart enough at the current version but it wasn't in the past, you'll have to do a direct export.

```js
export { default as Img } from "gatsby-image"
export { default as ImgWithIEPolyfill } from "gatsby-image/withIEPolyfill"
```

Reply:
Done